### ISSUE TITLE
Broken sources for europe / Poland

### ISSUE THREAD

**Issue by rbuffat** [osmlab/editor-layer-index#960]
Currently, there seem to be a few broken sources for Poland: https://rbuffat.github.io/eli_watchdog/ => europe => europe / pl
However, it is also possible that all or some of them are just geoblocked. 

It would be good if the existing sources could be reviewed if they are still available and they still can be used for OSM. 
@Asteliks You seem to be quite active, maybe you have some local knowledge or know somebody that has.

**Comment by Asteliks** [comment]
I can take a look at it but from what I can tell this will need me to contact the local authorities that are sharing these maps. In general this may take a while

**Comment by Asteliks** [comment]
also is there any way I can get my PR merged faster?

**Comment by andrewharvey** [comment]
> also is there any way I can get my PR merged faster?

any existing analysis or discussion about this license and if it's compatible with OSM would help

**Comment by jendrusk** [comment]
Let me know if (and how) can I help with this - I'm from Poland, I'm a President of a Board of Polish OSM Association and I'm a GIS Specialist.

**Comment by Asteliks** [comment]
For starters we will need a License URL and a Privacy Policy URL for all the maps used in Poland, then we will need to update the imagery URLs and give all the maps appropriate categories. @jendrusk Could you supply us with the appropriate legal URLs confirming that OSM and its users can use the available maps?

**Comment by rbuffat** [comment]
@jendrusk @Asteliks I just noticed that many sources in Poland seem to be not working. But it could also be that the tests are faulty or they could be geoblocked and only available from within Poland. 

If a source is really not available anymore it would be nice to see if there is a replacement (maybe there is a new url, or there is a replacement service). Also, it would be really good to document the license for each source. This can be achieved with the license_url attribute. It should ideally contain an URL that contains information about the license of the data. In the best case, it is possible to determine that the data can be used for OSM from the license information provided. If it is unclear, there is also possible to ask the data owners for explicit permission. 

The privacy_policy_url should contain an URL to a page with information about what data is stored about users accessing the data (cookies, etc.). With the GDPR most sites in the European Union should have such a page, however, there are still site without them. In most cases, there is no privacy policy directly for e.g. a wms server, but for the domain it belongs. E.g. there is no privacy policy for wms.geoportal.xy, but on geoportal.xy a policy can be found. 

All Polish sources are in the source/europe/pl directory: https://github.com/osmlab/editor-layer-index/tree/gh-pages/sources/europe/pl
The index for JOSM may contain already updates: https://josm.openstreetmap.de/wiki/Maps/Poland
It should just be noted that entries in this index are not reviewed for their license compatibility.


I'm happy to help if I can.

**Comment by Asteliks** [comment]
@jendrusk I have been told that OSM Polska has signed cooperation agreements with some of the local authorities. Could you share the applicable agreements here? It could make our lives a lot easier. Besides that, we need to start checking the available sources, do you have an UpToDate list of them at OSM Polska?

**Comment by Asteliks** [comment]
We may have a game changer when it comes to map licenses in Poland. Based on my call (https://github.com/osmlab/editor-layer-index/pull/976) there has been a change in the Polish law making all ortofotomaps in Poland open. See [poz. 782](https://isap.sejm.gov.pl/isap.nsf/download.xsp/WDU20200000782/O/D20200782.pdf) for details. I’ll read into it in the free time.

**Comment by Asteliks** [comment]
Basically this. [Source](http://isap.sejm.gov.pl/isap.nsf/download.xsp/WDU19890300163/U/D19890163Lj.pdf)
![image](https://user-images.githubusercontent.com/62251475/97864506-fa3a4a00-1d08-11eb-8490-23e3096a8fc8.png)

We may need someone more experienced to interpret this. @jendrusk?

**Comment by jendrusk** [comment]
Yes, that's right. We have a massive game changer when it comes to licences in Poland. Few years ago Mr. Izdebski became a Chief Surveyor of the Country (position of deputy minister), and he's well known in community as a great supporter of opening data. In April 2019, we had a meeting where we signed an agreement under which it follows that geodetic resources such as an orthophotomap and a database of topographic objects with an accuracy of 1: 10000 are recognized as official documents and as a result are not subject to licensing (as understood by the minister, they are included in the public domain). To confirm this, an appropriate clause was included in the agreement, and all services provided by its office (WMS, WMTS) were provided with an appropriate note which shows that they can be used in any way, including reproduction and creation of derivative works. A few months later, the minister submitted a draft act opening almost all geodetic resources. The act passed through the legislative path and entered into force a few months ago, so now officially most maps, orthophotomaps and numerical terrain models in Poland are open data.

BTW - @Asteliks as a background - a significant number of Polish officials do not share the opinion of the main surveyor of the country, they come from the previous regime or are representatives of previous generations, when maps were a strategic and top secret resources, so in order to raise the prestige of their position and the work they do, they're forcing position that they should remain so, and consequently, they are not very friendly to any open data and software communities. Today we had a phone call from one of the offices you contacted by "what right someone in our community dare to tell them that they have an invalid license on the site." Therefore, my request - if you encounter such a situation, write to us again, we will try to settle it with due care in contacts.

**Comment by Asteliks** [comment]
@jendrusk Well that’s kind of funny because today I had two phone calls one with Gliwice (https://github.com/osmlab/editor-layer-index/pull/976) where I have asked their director for the full license as we only have access to a simplified version and one with GEOPOZ addressing the license issues brought up by @andrewharvey in https://github.com/osmlab/editor-layer-index/pull/950. I do not remember accusing anyone of using an invalid license what is more both conversations where quite nice. Could you be more specific?

EDIT: There is a chance that someone else is responsible for that "what right [...]" call. See https://github.com/osmlab/editor-layer-index/pull/978
